public/app/plugins/panel/stat/module.tsx

```diff
@@ -87,6 +87,13 @@ export const plugin = new PanelPlugin<Options>(StatPanel)
             { value: BigValueJustifyMode.Center, label: 'Center' },
           ],
         },
+      })
+      .addBooleanSwitch({
+        path: 'showPercentChange',
+        name: 'Show percent change',
+        defaultValue: defaultOptions.showPercentChange,
+        category: mainCategory,
+        showIf: (config) => config.textMode !== BigValueTextMode.Name && config.textMode !== BigValueTextMode.None,
```

Comment:
I could see text mode name and percent change being useful

Reply:
Agree!